Changelog.md

```diff
@@ -10,6 +10,7 @@ In v1.3.1 leader elections will be done entirely using the Lease API and no long
 v1.3.0 is a safe transition version, using v1.3.0 can automatically complete the merging of election locks, and then you can safely upgrade to v1.3.1.
 
 Also, *important note*, with the Release of Kubernetes v1.25 we are dropping support for the legacy branches, 
+Also, *important note*, with the Release 1.25 we are dropping support for the legacy branches, 
```

Comment:
I still feel it will help to avoid ambiguity if we get explicit here and say `with the release of Kubernetes v1.25, we are dropping support for the legacy edition, that means all version 0.X of the ingress-nginx-controller`

Reply:
updated it